ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -463,7 +477,15 @@ generateHieAsts hscEnv tcm =
     -- don't export an interface which allows for additional information to be added to hie files.
     let fake_splice_binds = listToBag (map (mkVarBind unitDataConId) (spliceExpresions $ tmrTopLevelSplices tcm))
         real_binds = tcg_binds $ tmrTypechecked tcm
+#if MIN_VERSION_ghc(9,0,1)
+        -- TODO: Use some proper values here!
+        evBinds = emptyBag @EvBind :: Bag EvBind
```

Comment:
Please fix this, all the values come from the typechecked source:

```haskell
      top_ev_binds = tcg_ev_binds ts
      insts = tcg_insts ts
      tcs = tcg_tcs ts


```

Reply:
Thanks!